defaults/main.yml

```diff
@@ -10,6 +10,11 @@ icinga2_agent_apt:
   repo: "deb http://packages.icinga.com/{{ ansible_distribution|lower }} icinga-{{ ansible_distribution_release }} main"
   key: "http://packages.icinga.com/icinga.key"
 
+# The suse repository and key from Icinga
+icinga2_agent_suse:
+  repo: "https://packages.icinga.com/SUSE/ICINGA-release.repo"
+  key: "http://packages.icinga.com/icinga.key"
+
 # The apt repository for stretch-backports
 stretch_backports_apt:
```

Comment:
also this variable isn't prefixed as `icinga2_` and not following our best practices. Should be fixed in a different PR.